app/views/your-psp/_stripe.njk

```diff
@@ -94,6 +94,12 @@
  {% else %}
  {% if  enableStripeOnboardingTaskList %}
     <h1 class="govuk-heading-m govuk-!-margin-top-8">Information for Stripe</h1>
+    <h3 class="govuk-body"> {{progressIndicator.no_of_tasks_completed}} out of {{progressIndicator.total_number_of_tasks}} steps completed</h3>
```

Comment:
I don't think this should be a h3.
Are we following the guidance in the task list pattern? https://design-system.service.gov.uk/patterns/task-list-pages/

If so, looks like the progress indicator should be below the h2 in a `<p> `- as in this example https://prototype-kit.service.gov.uk/v12/docs/templates/task-list

Reply:
You're right, good spot